README.md

```diff
@@ -1,76 +1,39 @@
 # CUDA-Python
 
-CUDA Python is a standard set of low-level interfaces, providing full coverage of and access to the CUDA host APIs from Python. Checkout the [Overview](https://nvidia.github.io/cuda-python/overview.html) for the workflow and performance results.
+CUDA Python is the home for accessing NVIDIA’s CUDA platform from Python. It consists of multiple components:
```

Comment:
The distinction I was trying to follow where:
- "CUDA Python" refers to the ecosystem
- "CUDA-Python" (or "cuda-python") refers to this repo specifically.

> Could we find something more concrete to delineate the scope of CUDA-Python, so that readers aren't led to expect too much? 

The way I tried to communicate this is with:
```
CUDA-Python is structured to become a metapackage that contains a collection of subpackages.
```
and then listing each subpackage with their own section. Maybe this could come across cleaner if I strictly use "`cuda-python`" when referring to this repo, while using just "CUDA Python" while referring to the ecosystem?

Reply:
> CUDA-Python is structured to become a metapackage that contains a collection of subpackages.

Unfortunately I got sidetracked before I got to that part ...

I'd have suggested to put that at the top. That'd really make it clear right from the start.

> Maybe this could come across cleaner if I strictly use "cuda-python" when referring to this repo, while using just "CUDA Python" while referring to the ecosystem?

Yes, I really like that idea.

BTW it's completely fine with me if you leave that for later.

Reply:
Yes I already used `cuda-python` or `NVIDIA/cuda-python` to refer to this project/repo or packages generated from here. I'll take care of this.

Reply:
Updated in commit 30d65ed1ceb45a901742626587e1211c018d55b4.